modules/op-release-notes-1-19.adoc

```diff
@@ -0,0 +1,396 @@
+[source,yaml]
+----
+spec:
+  serviceAccountName: tekton-triggers-example-sa
+  resources:
+    kubernetesResource:
+      spec:
+        template:
+          spec:
+            securityContext:
+              runAsNonRoot: true
+            containers:
+            - resources:
+                requests:
+                  memory: "64Mi"
+                  cpu: "250m"
+                limits:
+                  memory: "128Mi"
+                  cpu: "500m"
+              securityContext:
+                readOnlyRootFilesystem: true
+----
+
+* With this update, you can configure custom database credentials for {tekton-results} using the `TektonConfig` custom resource (CR). This eliminates the need to rely on default PostgreSQL secrets that use default usernames and passwords.
+
+* With this update, {pac} no longer creates a pending status on GitHub Pull Requests when an unauthorized bot user attempts to trigger a `PipelineRun`. Instead of generating a blocking status check, such requests are now silently disallowed.
+
+* With this update, a new metric, `pipelines_as_code_git_provider_api_request_count`, tracks the number of API calls made by {pac} to Git providers, such as GitHub, GitLab, and Gitea. The metric also helps to monitor API rate limit usage across Git providers, namespaces, event types, and repositories.
+
+* With this update, repository URLs are now validated during creation to ensure they are properly formatted and use valid schemes, such as `http` or `https`. This enhancement helps prevent misconfigurations and runtime errors.
+
+* With this update, `PipelineRun` status comments now render correctly in markdown on Bitbucket Data Center and Bitbucket Cloud, instead of appearing as raw strings in the pull request UI.
+
+* With this update, when the `generateSigningSecret` key is enabled, the {pipelines-title} Operator automatically creates Cosign key pairs, specifically `cosign.key` (private key) and `cosign.pub` (public key). These key pairs are then stored within the `signing-secrets` secret, located in the `tekton-pipelines` namespace. You can enable the `generateSigningSecret` key by setting it to `true` in the `TektonConfig` custom resource.
++
+.Example of enabling Cosign key pairs
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  chain:
+    disabled: false
+    generateSigningSecret: true
+----
+
+* With this update, users can retrieve logs from Splunk that were forwarded by OpenShift Logging. To configure this feature, you can set environment variables such as `SPLUNK\_SEARCH\_TOKEN, LOGGING\_PLUGIN\_QUERY\_PARAMS` (including index details such as index=konflux), and `LOGGING\_PLUGIN\_API\_URL` (with the Splunk endpoint and port number).
+
+* With this update, the {tekton-results} watcher leverages `StatefulSet` ordinals for improved high availability and workload distribution as an alternative to leader election.
```

Comment:
the release notes say this is Tekton Chains but the example config is inside of the `chains` part of the Tekton Config. Is that correct?

Also, do i understand right that Tekton Results is part of a stateful set instead of a deployment? @khrm is that correct or is this a setting which can be enabled?

Reply:
@aThorp96 can you clarify your ask ?

Reply:
Probably, what's meant here is that we can set up run watcher (knative controller) as a statefulset with multiple replica.